wiki/Community/Project_Loved/en.md

```diff
@@ -48,7 +48,8 @@ There are no strict rules governing which maps a captain must nominate for Loved
 
 All of the organisation and planning for Project Loved is done in the `#osu-loved` channel of the [osu!dev Discord server](https://discord.gg/ppy). There is also a `#project-loved` channel in the [osu! Community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H). Any questions, concerns, or suggestions may be directed there.
 
-In addition to the discussion channels mentioned above, osu!taiko and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
+In addition to the discussion channels mentioned above, the osu!taiko, osu!catch, and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
```

Comment:
If this description is changed, the translations should either be updated accordingly, or get the outdated tag.